<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationDialog</class>
 <widget class="QDialog" name="InformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>196</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>521</width>
    <height>196</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>521</width>
    <height>196</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Device Information</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/active64.png</normaloff>
    <normalon>:/icons/icons/active64.png</normalon>
    <disabledoff>:/icons/icons/greyed64.png</disabledoff>
    <disabledon>:/icons/icons/greyed64.png</disabledon>
    <activeoff>:/icons/icons/active64.png</activeoff>
    <activeon>:/icons/icons/active64.png</activeon>
    <selectedoff>:/icons/icons/selected64.png</selectedoff>
    <selectedon>:/icons/icons/selected64.png</selectedon>:/icons/icons/active64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>155</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelManufacturer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Manufacturer:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelManufacturerValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelProduct">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>27</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Product:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelProductValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>27</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSerial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Serial number:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSerialValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>44</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelVID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>61</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Vendor ID:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelVIDValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>61</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>78</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Product ID:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPIDValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>78</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPowerMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Power mode:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPowerModeValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>95</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaxPower">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>112</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. power consumpt.:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaxPowerValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>112</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelRemoteWakeUpCapable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>129</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Remote wake-up capable:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRemoteWakeUpCapableValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>129</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InformationDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
